docs/docs-content/vertex/install-palette-vertex/install-on-kubernetes/install.md

```diff
@@ -710,51 +723,41 @@ helm upgrade --values spectro-mgmt-plane/values.yaml image-swap image-swap-0.0.0
 
    :::
 
-9. Use the custom domain name or the IP address of the load balancer to visit the VerteX system console. To access the
-   system console, open a web browser and paste the custom domain URL in the address bar and append the value `/system`.
-   Replace the domain name in the URL with your custom domain name or the IP address of the load balancer.
-   Alternatively, you can use the load balancer IP address with the appended value `/system` to access the system
-   console.
-
-   The first time you visit the VerteX system console, a warning message about a not-trusted SSL certificate may appear.
-   This is expected, as you still need to upload your SSL certificate to VerteX. You can ignore this warning message and
-   proceed.
-
-<br />
+10. Use the custom domain name or the IP address of the load balancer to visit the VerteX system console. To access the
+    system console, open a web browser and paste the custom domain URL in the address bar and append the value
+    `/system`. Replace the domain name in the URL with your custom domain name or the IP address of the load balancer.
+    Alternatively, you can use the load balancer IP address with the appended value `/system` to access the system
+    console.
 
-![Screenshot of the VerteX system console showing Username and Password fields.](/vertex_install-on-kubernetes_install_system-console.png)
+    The first time you visit the VerteX system console, a warning message about a not-trusted SSL certificate may
+    appear. This is expected, as you still need to upload your SSL certificate to VerteX. You can ignore this warning
```

Comment:
Do you think it would make sense to link to vertex/system-management/ssl-certificate-management.md here?

Reply:
@yuliiiah it's mentioned in step 11, so I think we're good